common-content-dataflow/modules/ROOT/partials/prerequisites.adoc

```diff
@@ -11,19 +11,80 @@ If you don't have an instance yet, you have two options:
 - sign-up for a free link:https://neo4j.com/cloud/aura-free/[AuraDB] instance
 - install and self-host Neo4j in a location that is publicly accessible (see link:https://neo4j.com/docs/operations-manual/current/installation/[Neo4j -> Installation]) with port 7687 open (Bolt protocol)
 
-Either way, you then need to create a file containing your database connection information in JSON format (basic authentication is the only supported method).
+Either way, you then need to create a file containing your database connection information in JSON format.
 This file will go into your Cloud Storage bucket.
 
-.neo4j-connection-info.json
-[source, JSON]
+[.tabbed-example]
```

Comment:
I cannot make the display of tabs work at all.
Would you know what I am missing @stefano-ottolenghi?

Reply:
Apparently the syntax for each tab is `[.include-with-textWithDashes]`, where `textWithDashes` is a `[^\s]*` string in which dashes get replaced by spaces. See https://github.com/neo4j-documentation/docs-ui/blob/master/src/js/08-tabs-block.js

Reply:
oh ok, thanks for looking into this!